### ISSUE TITLE
ZoL taking center stage

### ISSUE THREAD

**Issue by ylluminate** [openzfsonosx/zfs#674]
@lundman thought this was interesting:
https://www.phoronix.com/scan.php?page=news_item&px=FreeBSD-ZFS-On-Linux

**Comment by lundman** [comment]
Yeah, its amusing since we spent time looking at changing from Linux to illumos, but never pulled the trigger on it fully, and now most likely we will take the repo back to Linux.
